package.json

```diff
@@ -1,8 +1,16 @@
 {
+  "name": "subnet-assets",
+    "version": "1.0.0",
+    "description": "repo containing info for subnets in the Avalanche ecosystem",
+    "repository": "git@github.com:ava-labs/subnet-assets.git",
+    "author": "Dov Brand <dov.brand@avalabs.org>",
+    "license": "MIT",
+    "type": "module",
```

Comment:
We shouldn't fill out this stuff I think, makes it seem like we're going to publish this as an npm package which we will not do

Reply:
Should add `"private": true`

Reply:
Done